.github/workflows/create-release.yml

```diff
@@ -95,29 +96,42 @@ jobs:
       shell: cmd
       run: install_sct.bat
 
-    - name: Update environment variables
+    - name: Update environment variables (Unix)
+      if: matrix.os != 'windows-2019'
+      # NB: install_sct edits ~/.bashrc, but those environment changes don't get passed to subsequent steps in GH Actions.
+      # So, we filter through the .bashrc and pass the values to $GITHUB_ENV and $GITHUB_PATH.
+      # Relevant documentation: https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#environment-files
+      # This workaround should be replaced by https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3198#discussion_r568225392
       run: |
-        if [ "$RUNNER_OS" == "Windows" ]; then
-          # NB: I'm not sure what GHA's syntax is for cmd.exe, so we use bash just for this one change
-          # In a user install, the user would perform this step using the Windows PATH-changing GUI.
-          echo "$GITHUB_WORKSPACE/bin" >> $GITHUB_PATH
-        else
-          # NB: install_sct edits ~/.bashrc, but those environment changes don't get passed to subsequent steps in GH Actions.
-          # So, we filter through the .bashrc and pass the values to $GITHUB_ENV and $GITHUB_PATH.
-          # Relevant documentation: https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#environment-files
-          # This workaround should be replaced by https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3198#discussion_r568225392
-          cat ~/.bashrc | grep "export SCT_DIR" | cut -d " " -f 2 >> $GITHUB_ENV
-          cat ~/.bashrc | grep "export PATH" | grep -o "/.*" | cut -d ':' -f 1 >> $GITHUB_PATH
-        fi
+        cat ~/.bashrc | grep "export SCT_DIR" | cut -d " " -f 2 >> $GITHUB_ENV
+        cat ~/.bashrc | grep "export PATH" | grep -o "/.*" | cut -d ':' -f 1 >> $GITHUB_PATH
+
+    - name: Update environment variables (Windows)
+      if: matrix.os == 'windows-2019'
+      shell: cmd
+      # - NB1: In a user install, the user would perform this step using the Windows PATH-changing GUI.
+      # - NB2: Because `requirements-freeze.txt` is present, SCT gets installed to `~/sct_0.0`, rather
+      #   than an in-place install. So, we specify this path directly, rather than using $GITHUB_WORKSPACE.
+      run: |
+        echo %HOMEPATH%
+        echo %HOMEPATH%\sct_0.0\bin >> %GITHUB_PATH%
```

Comment:
I've found some strange behavior that I'm still struggling to wrap my head around:

- On `windows-2019` runners, inside `shell: cmd`, running `install_sct.bat`, SCT will be installed to `%HOMEPATH%\sct_%SCT_VERSION%`. 
- This gets evaluated to `\Users\runneradmin\sct_0.0`.
- Naturally then, to add the SCT scripts to the path, then, we would add `%HOMEPATH%\sct_0.0\bin` to the PATH. Yet, for some reason, calling an SCT script fails with a "script not found" error.

Investigation to follow.

Reply:
The first thing I noticed was that the path `~` differs depending on the shell you're in, and what variable you query:

- bash
  - `echo $HOME` returns `/home/runneradmin/`
  - `echo $HOMEPATH` returns `\Users\runneradmin\`
- cmd
  - `echo %HOME%` returns `C:\msys64\home\runneradmin`
  - `echo %HOMEPATH%` returns `\Users\runneradmin\`

Reply:
The second thing I realized is that, when running a `bash` shell, there seem to be _two separate drives_ in the filesystem for `windows-2019` GHA runners: `/c` and `/d`. These drives aren't visible when you `ls -h` inside `/`, but you can `cd` into them in a `tmate` session and poke around:

```console
runneradmin@fv-az276-936 MINGW64 /d
# ls 
'$RECYCLE.BIN'   a   cmds.txt   CollectGuestLogsTemp   DATALOSS_WARNING_README.txt   diskpartlog.txt   pagefile.sys   resizepartition.ps1  'System Volume Information'   Users 

runneradmin@fv-az276-936 MINGW64 /c 
# ls 
'$Recycle.Bin'   Android           Config.Msi                ghcup             imagedata.json   Microsoft   msys64   Packages         'Program Files'         Recovery   selenium             software-report.json  'System Volume Information'   Users     WindowsAzure 
 actions         cf-cli            data                      hostedtoolcache   inetpub          Miniconda   mysql    PerfLogs         'Program Files (x86)'   rtools40   SeleniumWebDrivers   software-report.md     Temp                         vcpkg 
 aliyun-cli      cobertura-2.1.1  'Documents and Settings'   image             Julia            Modules     npm      post-generation   ProgramData            runners    shells               Strawberry             tools                        Windows
```

Digging deeper, SCT gets installed to `/d/Users/runneradmin`, despite the fact that this folder is otherwise bare. (The "full" user folder is `/c/Users/runneradmin/`.)

Reply:
The third thing I noticed is that paths starting with `/` that don't include a drive letter are treated as paths _relative to the drive you're currently on_.

 - `cd /c && cd $HOMEPATH` will bring you to `/c/Users/runneradmin`
 - `cd /d && cd $HOMEPATH` will bring you to `/d/Users/runneradmin`

Reply:
All this is to say that, I think when adding `%HOMEPATH%\sct_0.0\bin` to the path, we need to go a step further and be explicit about the drive letter?

Reply:
This didn't work... I'm a bit of a loss here. :sweat:

Reply:
Ah! It worked! 

`sct_check_dependencies` runs using `echo "/d/$HOMEPATH/sct_0.0/bin" >> $GITHUB_PATH`.

Reply:
Ah, okay.  So:

- Before this PR, we used `$GITHUB_WORKSPACE/bin`, which evaluates to `/d/a/spinalcordtoolbox/spinalcordtoolbox/bin`
- After this PR, we now use `/d$HOMEPATH/sct_0.0/bin`, which evaluates to `/d/Users/runneradmin/sct_0.0/bin`

If we add both of these to the `PATH`:

<details>
<summary> echo $PATH </summary>

```
/mingw64/bin:/usr/bin:/c/Users/runneradmin/bin:/d/Users/runneradmin/sct_0.0/bin:/d/a/spinalcordtoolbox/spinalcordtoolbox/bin:/c/Program Files/MongoDB/Server/5.0/bin:/c/aliyun-cli:/c/vcpkg:/c/cf-cli:/c/Program Files (x86)/NSIS:/c/tools/zstd:/c/Program Files/Mercurial:/c/hostedtoolcache/windows/stack/2.9.3/x64:/c/cabal/bin:/c/ghcup/bin:/c/Program Files/dotnet:/c/mysql/bin:/c/Program Files/R/R-4.2.2/bin/x64:/c/SeleniumWebDrivers/GeckoDriver:/c/Program Files (x86)/sbt/bin:/c/Program Files (x86)/GitHub CLI:/bin:/c/Program Files (x86)/pipx_bin:/c/npm/prefix:/c/hostedtoolcache/windows/go/1.17.13/x64/bin:/c/hostedtoolcache/windows/Python/3.7.9/x64/Scripts:/c/hostedtoolcache/windows/Python/3.7.9/x64:/c/hostedtoolcache/windows/Ruby/2.5.9/x64/bin:/c/tools/kotlinc/bin:/c/hostedtoolcache/windows/Java_Temurin-Hotspot_jdk/8.0.362-9/x64/bin:/c/Program Files/ImageMagick-7.1.0-Q16-HDRI:/c/Program Files (x86)/Microsoft SDKs/Azure/CLI2/wbin:/c/ProgramData/kind:/c/Program Files/Eclipse Foundation/jdk-8.0.302.8-hotspot/bin:/c/Windows/system32:/c/Windows:/c/Windows/System32/Wbem:/c/Windows/System32/WindowsPowerShell/v1.0:/c/Windows/System32/OpenSSH:/c/ProgramData/Chocolatey/bin:/c/Program Files/PowerShell/7:/c/Program Files/Microsoft/Web Platform Installer:/c/Program Files/Microsoft SQL Server/130/Tools/Binn:/c/Program Files/Microsoft SQL Server/Client SDK/ODBC/170/Tools/Binn:/c/Program Files (x86)/Windows Kits/10/Windows Performance Toolkit:/c/Program Files (x86)/Microsoft SQL Server/110/DTS/Binn:/c/Program Files (x86)/Microsoft SQL Server/120/DTS/Binn:/c/Program Files (x86)/Microsoft SQL Server/130/DTS/Binn:/c/Program Files (x86)/Microsoft SQL Server/140/DTS/Binn:/c/Program Files (x86)/Microsoft SQL Server/150/DTS/Binn:/c/Program Files (x86)/Microsoft SQL Server/160/DTS/Binn:/c/Program Files/OpenSSL/bin:/c/Strawberry/c/bin:/c/Strawberry/perl/site/bin:/c/Strawberry/perl/bin:/c/ProgramData/chocolatey/lib/pulumi/tools/Pulumi/bin:/c/Program Files/TortoiseSVN/bin:/c/Program Files/CMake/bin:/c/ProgramData/chocolatey/lib/maven/apache-maven-3.9.0/bin:/c/Program Files/Microsoft Service Fabric/bin/Fabric/Fabric.Code:/c/Program Files/Microsoft SDKs/Service Fabric/Tools/ServiceFabricLocalClusterManager:/c/Program Files/nodejs:/cmd:/mingw64/bin:/usr/bin:/c/Program Files/GitHub CLI:/c/tools/php:/c/Program Files (x86)/sbt/bin:/c/SeleniumWebDrivers/ChromeDriver:/c/SeleniumWebDrivers/EdgeDriver:/c/Program Files/Amazon/AWSCLIV2:/c/Program Files/Amazon/SessionManagerPlugin/bin:/c/Program Files/Amazon/AWSSAMCLI/bin:/c/Program Files (x86)/Google/Cloud SDK/google-cloud-sdk/bin:/c/Program Files (x86)/Microsoft BizTalk Server:/c/Program Files/LLVM/bin:/c/Users/runneradmin/.dotnet/tools:/c/Users/runneradmin/.cargo/bin:/c/Users/runneradmin/AppData/Local/Microsoft/WindowsApps
```

Specifically:

```
/d/Users/runneradmin/sct_0.0/bin:/d/a/spinalcordtoolbox/spinalcordtoolbox/bin
```

</details>

Then check what the `PATH` is _within the context of `sct_check_dependences`_:

<details>
<summary> print(f"\n{os.environ['PATH']}\n") </summary>

```
D:\Users\runneradmin\sct_0.0\python\envs\venv_sct\lib\site-packages\PyQt5\Qt\bin;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Users\runneradmin\bin;/d\Users\runneradmin/sct_0.0/bin;D:\a\spinalcordtoolbox\spinalcordtoolbox/bin;C:\Program Files\MongoDB\Server\5.0\bin;C:\aliyun-cli;C:\vcpkg;C:\cf-cli;C:\Program Files (x86)\NSIS\;C:\tools\zstd;C:\Program Files\Mercurial\;C:\hostedtoolcache\windows\stack\2.9.3\x64;C:\cabal\bin;C:\\ghcup\bin;C:\Program Files\dotnet;C:\mysql\bin;C:\Program Files\R\R-4.2.2\bin\x64;C:\SeleniumWebDrivers\GeckoDriver;C:\Program Files (x86)\sbt\bin;C:\Program Files (x86)\GitHub CLI;C:\Program Files\Git\bin;C:\Program Files (x86)\pipx_bin;C:\npm\prefix;C:\hostedtoolcache\windows\go\1.17.13\x64\bin;C:\hostedtoolcache\windows\Python\3.7.9\x64\Scripts;C:\hostedtoolcache\windows\Python\3.7.9\x64;C:\hostedtoolcache\windows\Ruby\2.5.9\x64\bin;C:\tools\kotlinc\bin;C:\hostedtoolcache\windows\Java_Temurin-Hotspot_jdk\8.0.362-9\x64\bin;C:\Program Files\ImageMagick-7.1.0-Q16-HDRI;C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2\wbin;C:\ProgramData\kind;C:\Program Files\Eclipse Foundation\jdk-8.0.302.8-hotspot\bin;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Windows\System32\OpenSSH\;C:\ProgramData\Chocolatey\bin;C:\Program Files\PowerShell\7\;C:\Program Files\Microsoft\Web Platform Installer\;C:\Program Files\Microsoft SQL Server\130\Tools\Binn\;C:\Program Files\Microsoft SQL Server\Client SDK\ODBC\170\Tools\Binn\;C:\Program Files (x86)\Windows Kits\10\Windows Performance Toolkit\;C:\Program Files (x86)\Microsoft SQL Server\110\DTS\Binn\;C:\Program Files (x86)\Microsoft SQL Server\120\DTS\Binn\;C:\Program Files (x86)\Microsoft SQL Server\130\DTS\Binn\;C:\Program Files (x86)\Microsoft SQL Server\140\DTS\Binn\;C:\Program Files (x86)\Microsoft SQL Server\150\DTS\Binn\;C:\Program Files (x86)\Microsoft SQL Server\160\DTS\Binn\;C:\Program Files\OpenSSL\bin;C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bin;C:\ProgramData\chocolatey\lib\pulumi\tools\Pulumi\bin;C:\Program Files\TortoiseSVN\bin;C:\Program Files\CMake\bin;C:\ProgramData\chocolatey\lib\maven\apache-maven-3.9.0\bin;C:\Program Files\Microsoft Service Fabric\bin\Fabric\Fabric.Code;C:\Program Files\Microsoft SDKs\Service Fabric\Tools\ServiceFabricLocalClusterManager;C:\Program Files\nodejs\;C:\Program Files\Git\cmd;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Program Files\GitHub CLI\;c:\tools\php;C:\Program Files (x86)\sbt\bin;C:\SeleniumWebDrivers\ChromeDriver\;C:\SeleniumWebDrivers\EdgeDriver\;C:\Program Files\Amazon\AWSCLIV2\;C:\Program Files\Amazon\SessionManagerPlugin\bin\;C:\Program Files\Amazon\AWSSAMCLI\bin\;C:\Program Files (x86)\Google\Cloud SDK\google-cloud-sdk\bin;C:\Program Files (x86)\Microsoft BizTalk Server\;C:\Program Files\LLVM\bin;C:\Users\runneradmin\.dotnet\tools;C:\Users\runneradmin\.cargo\bin;C:\Users\runneradmin\AppData\Local\Microsoft\WindowsApps
```

Notably:

```
/d\Users\runneradmin/sct_0.0/bin;D:\a\spinalcordtoolbox\spinalcordtoolbox/bin
```

We see:

- `D:\a\spinalcordtoolbox\spinalcordtoolbox/bin`
- `/d\Users\runneradmin/sct_0.0/bin`

So, Python is smart enough to change he _old_ path from `/d` to `D:`, but it keeps the new path in the (incorrect?) form, which is surely causing the FileNotFoundError.

But... why???

Reply:
That said, on a whim I tried setting `is_sct_binary=True` in the call to `run_proc`, and it seems to fix the PATH issues entirely.

Given that this entire thread is devoted to working around quirks specific to GitHub Actions, I'm kind of just fine... not even bothering trying to find a better solution? I don't want to have to think about how Python interprets the GHA filesystem anymore.